packages/gestalt/src/Video/Playhead.js

```diff
@@ -30,9 +30,14 @@ export default class VideoPlayhead extends PureComponent<Props, State> {
   seek: (clientX: number) => void = (clientX) => {
     if (this.playhead) {
       const { duration, seek } = this.props;
-      const { left, width } = this.playhead.getBoundingClientRect();
-      const percent = Math.max(0, Math.min((clientX - left) / width, 1));
+      const { left, right, width } = this.playhead.getBoundingClientRect();
+
+      const isRTL = window.getComputedStyle(this.playhead).direction === 'rtl';
```

Comment:
my concern here is that if this is server-rendered there's no window

you probably have to do `if (window) {}`

Reply:
Why Math.abs()?, that's a new calculation here. jut curious

Reply:
Valid concern. I overlooked that because I thought to myself "what are the chances of `seek()` method being called on the server-side.

Reply:
But still it is good to be on the safe side. I will add a check 👍